packages/common/models/dynamicLists-core.model.ts

```diff
@@ -16,12 +16,13 @@
 
 import { BeagleUIElement } from '@zup-it/beagle-web'
 import { BeagleComponent } from '../types'
-import { Direction, BeagleDefaultComponent } from './types'
+import { BeagleDefaultComponent, Direction } from './types'
 
 export type NodeType = HTMLElement | null
 
-export interface BeagleFutureListViewInterface extends BeagleDefaultComponent, BeagleComponent {
-  direction: Direction,
+export type ListType = 'GRID' | 'LIST'
```

Comment:
On TypeScript, we are able to create [String Enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums), so would be a cleaner approach to use on the application, avoiding some casts using `as` and is easier to do maintenance and for new types of dynamic list. Other benefits are the ability to quickly find all references to the list type on the whole project structure and it is also really quicker to write, because using the auto-complete of your editor you do not have to worry about misspelling the string or validating it.

I suggest implement this (also valid for Direction) like this:

```suggestion
export enum ListType {
  Grid='GRID',
  List='LIST'
}

export enum ListDirection {
  Horizontal='HORIZONTAL',
  Vertical='VERTICAL'
}
```

Then, when you would use this enum, you simply use like this:
```javascript
listType: ListType.Grid
// OR
listType: ListType.List
```

Reply:
As Tiago pointed out on this PR https://github.com/ZupIT/beagle-web-angular/pull/245. We avoid using enums on the projects so I am resolving the related requests